[docs]yugabyted: Updating example of `Pass additional flags to YB-Master and YB-TServer` in yugabyted (#25717)
yugabyte/yugabyte-db: docs/content/preview/reference/configuration/yugabyted.md

./bin/yugabyted start --tserver_flags="pg_yb_session_timeout_ms=1200000,ysql_max_connections=400"
```

> When setting CSV value flags, such as [--ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv), you need to enclose the values inside curly braces `{}`. If a setting include double quotes (`"`), these double quotes should be preceded by a backslash (`\`) to make it an escape sequence. For example:

Reviewer: ```suggestion
When setting CSV value flags, such as [--ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv), you need to enclose the values inside curly braces `{}`; if a setting includes double quotes (`"`), precede the double quotes with a backslash (`\`) to make it an escape sequence. For example:
```
